docs/topics/auth/customizing.txt

```diff
@@ -850,24 +853,6 @@ if those assumptions are met:
   default) that can be used to identify the user and a boolean field named
   ``is_active`` to prevent password resets for inactive users.
 
-Finally, the following forms are tied to
```

Comment:
Most likely the example still has value as there are still use cases for extending it.

Reply:
Maybe we could chang to something like `If you still need customization in this form`?

Reply:
Right, some small word changes are needed.